### PR TITLE
Interpreters can cancel translation

### PR DESCRIPTION
**Pull request by hackermariama**

### CODE REVIEW

File: client/src/api/translations.js [hunk 1]
```diff
@@ -13,6 +13,21 @@ export const pickDocument = id => {
   return fetch("/api/translations", userRequest);
 };
 
+export const cancelDocument = id => {
```
Comment (studio-sherpa) [inline client/src/api/translations.js]: The responsibility of this function is to cancel a translation so the name should be cancelTranslation and not cancelDocument

File: client/src/components/Dashboard.js [hunk 2]
```diff
@@ -42,6 +43,24 @@ class Dashboard extends Component {
         );
       });
   };
+  handleCancelDocumentClick = id => {
```
Comment (studio-sherpa) [inline client/src/components/Dashboard.js]: this handler function should be handleCancelTranslationClick

File: client/src/components/Dashboard.js [hunk 3]
```diff
@@ -103,6 +122,7 @@ class Dashboard extends Component {
                       userName={userName}
                       userRole={userRole}
                       handlePickDocumentClick={this.handlePickDocumentClick}
+                      handleCancelDocumentClick={this.handleCancelDocumentClick}
```
Comment (studio-sherpa) [inline client/src/components/Dashboard.js]: the key of this prop should be handleCancelTranslationClick

File: client/src/components/ActionColumn.js [hunk 4]
```diff
@@ -27,6 +27,9 @@ const ActionColumn = props => {
         <Link to={`/add-document-translation/${id}`}>
           <span>Submit Translation</span>
         </Link>
+        <Button onClick={() => props.handleCancelDocumentClick(id)}>
```
Comment (studio-sherpa) [inline client/src/components/ActionColumn.js]: The server endpoint that you are targeting is translations/id where the id is the translations id. 

On this file (ActionColumn), the variable id represents the document id and not the translations id.

You need to call props.handleCancelTranslationClick() with the id of the translation

**Review by llh1 — APPROVED** [review]

**Pull request merged**